pkg/api/server.go

```diff
@@ -770,19 +886,13 @@ func (h handler) getAPIsNamespaceResource(w http.ResponseWriter, r *http.Request
 		return
 	}
 
-	decoded, gvk, err := sbctl.Decode(resource, data)
+	decoded, _, err := sbctl.Decode(resource, data)
 	if err != nil {
 		log.Println("failed to decode wrapped", resource, ":", err)
 		w.WriteHeader(http.StatusInternalServerError)
 		return
 	}
 
-	if gvk.Group != group && gvk.Version != version {
```

Comment:
Does this check no longer work?

Reply:
for incoming ingress requests, 
URL:  `/apis/extensions/v1beta1/namespaces/default/ingresses`   
group would be: `extensions`
and response group would be `networking.k8s.io` .
version can differ b/w `v1` or `v1beta1`

In this scenario, it wont work [this applies to any k8s api-extensions]

Reply:
ok, looks like a hack, but it's not that important.